src/Updater.php

```diff
@@ -498,7 +500,7 @@ protected function validatePhar($phar)
 
     protected function cleanupAfterError()
     {
-        //@unlink($this->getBackupPharFile());
+        @unlink($this->getBackupPharFile());
```

Comment:
Moved the restoration to the cleanupAfterError function to make it a bit easier to understand why/when called.